### ISSUE TITLE
change the display position

### ISSUE THREAD

**Issue by ricky52be7** [John-Lluch/SWRevealViewController#521]
Is it possible to show the view from bottom to top with a slide up action? Please help.

**Comment by iDevelopper** [comment]
No, there is three controllers within SWRevealViewController (rear, front and right). You should implement this feature yourself with a pan gesture recognizer.

**Comment by ricky52be7** [comment]
thank you for your help.

**Comment by iDevelopper** [comment]
https://github.com/iDevelopper/PBRevealViewController
